Sample_v3/AppDelegate.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 import UIKit
-@testable import StreamChatClient_v3
+@testable import StreamChatClient
```

Comment:
Wait, this is not correct. We didn't change the name of the project, did we?? @buh

Reply:
It’s a new project the target count as a final name. Then on release we will only rename project itself.